tests/run/default_args_introspection.pyx

```diff
@@ -0,0 +1,19 @@
+# mode: run
```

Comment:
I think these tests should go in `cyfunction_defaults`

https://github.com/cython/cython/blob/efdf996dced075682d86b7e53a12b56b5ebfea38/tests/run/cyfunction_defaults.pyx

Reply:
Will do